src/pages/task/actionButtons/TaskNotificationModal.tsx

```diff
@@ -86,6 +86,12 @@ export const triggers: Trigger[] = [
     resourceType: "TASK",
     payloadResourceIdKey: "id",
   },
+  {
+    trigger: "task-failed-or-blocked",
+    label: "this task fails or is blocked",
```

Comment:
nit: can we capitalize the first letter to match the other labels

Reply:
I'm so glad you caught this it would've upset me so much